src/main/webapp/chart.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <script
+      type="text/javascript"
```

Comment:
Minor note: the `type` tag is no longer required in HTML5; if `type` is unset, it defaults to `text/javascript`.